apps/coscrad-frontend/src/components/tree-of-knowledge/category-tree.presenter.tsx

```diff
@@ -15,6 +15,8 @@ const wrapTree = ({
             <Accordion>
                 <AccordionSummary>{label}</AccordionSummary>
                 <AccordionDetails>
+                    {/* TODO Pass the state in as props */}
```

Comment:
Do this.

Reply:
Let's leave the category tree be. We are not really ready to present \ use it yet, anyway. We can refactor later.

Reply:
[Here](https://www.pivotaltracker.com/story/show/184310523) is a story for that work.